class.jetpack.php

```diff
@@ -689,7 +689,13 @@ private function __construct() {
 		add_filter( 'jetpack_get_default_modules', array( $this, 'handle_deprecated_modules' ), 99 );
 
 		// A filter to control all just in time messages
-		add_filter( 'jetpack_just_in_time_msgs', '__return_true', 9 );
+		add_filter( 'jetpack_just_in_time_msgs', function( $display ) {
+			if ( ! \Jetpack::is_active() || \Jetpack::is_development_mode() ) {
+				return false;
+			}
+            return true;
```

Comment:
Don't we want to return $display if we don't return false in case some other callback added to the filter is making it return false?

Reply:
it was returning`true` before 🤷‍♀️